pallets/corporate-actions/src/distribution/mod.rs

```diff
@@ -160,7 +162,8 @@ decl_module! {
         /// - `ca_id` identifies the CA to start a capital distribution for.
         /// - `portfolio` specifies the portfolio number of the CAA to distribute `amount` from.
         /// - `currency` to withdraw and distribute from the `portfolio`.
-        /// - `amount` of `currency` to withdraw and distribute.
+        /// - `per_share` amount of `currency` to withdraw and distribute.
```

Comment:
`per_share` must accommodate for 6 decimal places to allow for fractional values.